projects/prometheus/node_exporter/Makefile

```diff
@@ -0,0 +1,37 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG:=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=node_exporter
+REPO_OWNER=prometheus
+
+BINARY_TARGET_FILES=node_exporter
+EXTRA_GOBUILD_FLAGS=-tags netgo,osusergo,static_build
+
+NODE_EXPORTER_IMAGE_COMPONENT=prometheus/node-exporter
+
+EXCLUDE_FROM_STAGING_BUILDSPEC=true
+
+include $(BASE_DIRECTORY)/Common.mk
+
```

Comment:
on the topic of minimal base images, seems like this could work with one

Reply:
Updated